packages/react-components/react-table/stories/Table/Virtualized.stories.tsx

```diff
@@ -0,0 +1,236 @@
+  TableCellLayout,
+  createColumn,
+  Virtualizer,
+  useTableSort,
+} from '@fluentui/react-components/unstable';
+import { makeStyles } from '@fluentui/react-components';
+
+type FileCell = {
+  label: string;
+  icon: JSX.Element;
+};
+
+type LastUpdatedCell = {
+  label: string;
+  timestamp: number;
+};
+
+type LastUpdateCell = {
+  label: string;
+  icon: JSX.Element;
+};
+
+type AuthorCell = {
+  label: string;
+  status: PresenceBadgeStatus;
+};
+
+type Item = {
+  file: FileCell;
+  author: AuthorCell;
+  lastUpdated: LastUpdatedCell;
+  lastUpdate: LastUpdateCell;
+};
+
+const items: Item[] = [
+  {
+    file: { label: 'Meeting notes', icon: <DocumentRegular /> },
+    author: { label: 'Max Mustermann', status: 'available' },
+    lastUpdated: { label: '7h ago', timestamp: 3 },
+    lastUpdate: {
+      label: 'You edited this',
+      icon: <EditRegular />,
+    },
+  },
+  {
+    file: { label: 'Thursday presentation', icon: <FolderRegular /> },
+    author: { label: 'Erika Mustermann', status: 'busy' },
+    lastUpdated: { label: 'Yesterday at 1:45 PM', timestamp: 2 },
+    lastUpdate: {
+      label: 'You recently opened this',
+      icon: <OpenRegular />,
+    },
+  },
+  {
+    file: { label: 'Training recording', icon: <VideoRegular /> },
+    author: { label: 'John Doe', status: 'away' },
+    lastUpdated: { label: 'Yesterday at 1:45 PM', timestamp: 2 },
+    lastUpdate: {
+      label: 'You recently opened this',
+      icon: <OpenRegular />,
+    },
+  },
+  {
+    file: { label: 'Purchase order', icon: <DocumentPdfRegular /> },
+    author: { label: 'Jane Doe', status: 'offline' },
+    lastUpdated: { label: 'Tue at 9:30 AM', timestamp: 1 },
+    lastUpdate: {
+      label: 'You shared this in a Teams chat',
+      icon: <PeopleRegular />,
+    },
+  },
+];
+
+const repeatCount = 2500;
+const generateContent = (): Item[] => {
+  const contentList: Item[] = [];
+  for (let i = 0; i < repeatCount; i++) {
+    items.forEach((item, index) => {
+      const newItem: Item = {
+        file: { ...item.file },
+        author: { ...item.author },
+        lastUpdated: { ...item.lastUpdated },
+        lastUpdate: { ...item.lastUpdate },
+      };
+      // Update labels to make unique
+      newItem.author.label = `${newItem.author.label}-${i}-${index}`;
+      newItem.file.label = `${newItem.file.label}-${i}-${index}`;
+      newItem.lastUpdate.label = `${newItem.lastUpdate.label}-${i}-${index}`;
+      newItem.lastUpdated.label = `${newItem.lastUpdated.label}-${i}-${index}`;
+      contentList.push(newItem);
+    });
+  }
+  return contentList;
+};
+
+const fullItemList: Item[] = generateContent();
+
+// Simple wrapper class to control scroll view height
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    overflowAnchor: 'none',
+    overflowY: 'auto',
+    width: '100%',
+    height: '100%',
+    maxHeight: '750px',
+  },
+});
+
+export const Virtualized = () => {
```

Comment:
There is already a virtualization example in the docs, let's do some more work to distinguish them with some kind of naming scheme like
- Virtualization: react-window
- Virtualization: fluent virtualizer

You can put special characters in the name by doing something like this

https://github.com/microsoft/fluentui/blob/a2669d5c922e29311b92660bd4a4d3802a6dffd4/packages/react-components/react-button/stories/SplitButton/SplitButtonSizeSmall.stories.tsx#L94

Reply:
I would propose adding the example in a separate PR to make the changelist and required reviewers a bit smaller